lib/logstash/filters/jdbc/column.rb

```diff
@@ -0,0 +1,43 @@
+require_relative "validatable"
+
+module LogStash module Filters module Jdbc
+  class Column < Validatable
+    attr_reader :name, :datatype
+
+    private
+
+    def post_initialize
+      if valid?
+        @name = @name.to_sym
+        @datatype = @datatype.to_sym
+      end
+    end
+
+    def to_array
+      [@name.to_s, @datatype.to_s]
+    end
+
+    def parse_options
+      parsed = true
```

Comment:
Do you need the parsed variable? Or can you just use `@valid = @option_errors.empty?`